src/EditorFeatures/Core/PlatformCommands/GoToImplementationCommandArgs.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Editor.Commanding;
+
+namespace Microsoft.CodeAnalysis.Editor.PlatformCommands
```

Comment:
What does this namespace mean?

Reply:
This is not good actually. This is a new command args for a Roslyn own command (not shared at the editor level). It's here to stay, but the old one (Microsoft.CodeAnalysis.Editor.Commands.GoToImplementationCommandArgs) is going to go away eventually. The question is in which namespace do you want to have your own new command args. Microsoft.CodeAnalysis.Editor.Commanding.Commands perhaps?

I could delete Microsoft.CodeAnalysis.Editor.Commands.GoToImplementationCommandArgs, but I wanted to keep old commanding infrastructure intact, in case TS/F#/etc actually handle that command.

Reply:
Ok, I moved these into Microsoft.CodeAnalysis.Editor.Commanding.Commands namespace. New commands will be added in this namespace too (as opposed to old commands being in Microsoft.CodeAnalysis.Editor.Commands namespace).

---
In reply to: [159544643](https://github.com/dotnet/roslyn/pull/23769#discussion_r159544643) [](ancestors = 159544643)